content/applications/inventory_and_mrp/inventory/shipping/operation/multipack.rst

```diff
@@ -1,84 +1,119 @@
+:menuselection:`Inventory --> Delivery Orders`, then select a delivery order that has multiple
+items, a multiple quantity of the same item, or both.
 
-Sale order
-----------
+On the :guilabel:`Operations` tab, select the :guilabel:`⁞≣` menu icon in the line of the product
+that will be shipped in the first package.
 
-.. image:: multipack/multipack06.png
+.. image:: multipack/product-menu-icon.png
    :align: center
+   :alt: The menu icon for a product in a delivery order.
 
-Click on a **Delivery Method** to choose the right one.
+This makes a :guilabel:`Detailed Operations` pop-up window appear. In the table at the bottom of the
+pop-up window, the :guilabel:`Reserved` column shows the total quantity of the product included in
+the delivery order.
 
-The price is computed when you **save** the sale order or when you click on
-**Set price**.
+If the full quantity will be shipped in the first package, enter the same number in the
+:guilabel:`Done` column as appears in the :guilabel:`Reserved` column. If less than the full
+quantity will be shipped in the first package, enter a smaller number than the one that appears in
+the :guilabel:`Reserved` column. Click :guilabel:`Confirm` to confirm the :guilabel:`Done`
+quantities and close the pop-up.
 
-Multi-packages Delivery
------------------------
+.. image:: multipack/detailed-operations.png
+   :align: center
+   :alt: The Detailed Operations pop-up for a product in a delivery order.
 
-The delivery created from the sale order will take the shipping
-information from it.
+Repeat the same steps for every item quantity that is included in the first package. Then, click
+:guilabel:`Put In Pack` to create a package with all of the selected items.
 
-.. image:: multipack/multipack07.png
+.. image:: multipack/put-in-pack.png
    :align: center
+   :alt: The Put In Pack button on a delivery order.
 
-From here, you can split your delivery into multiple packages.
+For the next package, follow the same steps as detailed above, marking the quantity of each item to
+be included in the package as :guilabel:`Done` before clicking :guilabel:`Put In Pack` on the
+delivery order. Continue doing so until the full quantity of all items are added to a package.
 
-Choose the quantity you want to put in the first pack in the **Done**
-column, then click on the link **Put in Pack**.
+Finally, after all of the packages have been shipped, click :guilabel:`Validate` to confirm that the
+delivery order has been completed.
 
-.. image:: multipack/multipack02.png
+.. image:: multipack/validate-button.png
    :align: center
+   :alt: The Validate button on a delivery order.
 
-It will automatically create a pack with the requested quantity.
+.. tip::
+   After one or more packages are created, a :guilabel:`Packages` smart button appears in the
+   top-right corner of the delivery order. Click the :guilabel:`Packages` smart button to go to the
```

Comment:
I'd like to keep this as a GUI label since it's a button that can be clicked. I asked Melanie for her opinion and she agreed it should have the label.

Reply:
Yes after reading this again I agree that the GUI label is appropriate, sorry for the incorrect suggestion!